tests/with-docker/Earthfile

```diff
@@ -1,8 +1,9 @@
-VERSION --parallel-load 0.6
+VERSION --parallel-load --shell-out-anywhere 0.6
```

Comment:
Is it safe to include `--shell-out-anywhere` here, or should I move the docker tests into the `tests/shell-out-anywhere` directory to ensure with-docker tests run under the old expandArgs code?